src/pybkgmodel/data.py

```
@@ -501,23 +543,235 @@ def load_events(cls, file_name, cuts):
+                                              evt_head['MJDREFF'],
+                                              scale=evt_head['TIMESYS'].lower(),
+                                              format='mjd'
+                                              )
+
+                evt_time = evt_data['TIME'].quantity + ref_epoch
+                event_data['mjd'] = evt_time.mjd * u.d
+
+                # TODO: current observatory location only La Palma, no mandatory header keyword
+                obs_loc  = EarthLocation(lat=28.761758*u.deg,
+                                        lon=-17.890659*u.deg,
+                                        height=2200*u.m)
+
+                if evt_head['OBS_MODE'] in ('POINTING', 'WOBBLE'):
+
+                    alt_az_frame = AltAz(obstime=evt_time,
+                                        location=obs_loc)
+
+                    coords = SkyCoord(evt_head['RA_PNT'] *u.deg,
+                                    evt_head['DEC_PNT'] *u.deg,
+                                    frame='icrs')
+
+                    altaz_pointing =  coords.transform_to(alt_az_frame)
+
+                    event_data['pointing_zd'] = 90 * u.deg - altaz_pointing.alt
+                    event_data['pointing_az'] = altaz_pointing.az.to(u.deg)
+
+
+                    event_data['pointing_ra'] = np.array(
+                                                            [evt_head['RA_PNT']] \
+                                                            * len(event_data['pointing_zd'])
+                                                            ) * u.deg
+                    event_data['pointing_dec'] = np.array(
+                                                            [evt_head['DEC_PNT']] \
+                                                            * len(event_data['pointing_zd'])
+                                                            ) * u.deg
+
+                elif evt_head['OBS_MODE'] == 'DRIFT':
```

Just raise an error to let users know this is not supported instead of providing preliminary or invalid data (same for the next else block, this should be an actual error, not a print statement)

---

Drift mode is now tested, so no need for an error there. I however changed it for the case the file contains an unsupported mode. This will now raise an error instead of giving a warning.